app/views/lobby/home.scala

```diff
@@ -166,7 +166,9 @@ object home:
     trans.daysPerTurn,
     trans.ratingRange,
     trans.nbPlayers,
+    trans.nbPlayersOnline,
     trans.nbGamesInPlay,
+    trans.nbTotalGames,
```

Comment:
what's wrong with `nbPlayers` and `nbGamesInPlay`? It's what we currently use and it's correct?
I especially don't understand the total games thing. We're showing the number of games being played only

Reply:
I used those strings until I moved counters to be in the same place in both 3col and 4col and aligned them with the app tab controls (Quick Pairing/Lobby/Correspondence).
![image](https://github.com/lichess-org/lila/assets/101470903/7e671508-da02-40f6-82bf-dd65b47a7a84)
This was good but it created a bit of confusion with the number of correspondence games - so I figured some slightly more specific translations could help

Reply:
imo that's not an issue that wording tweaks can fix, especially since it will be translated.

indeed the line of text is not ideal, surely another solution can be found.